cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -16,16 +15,7 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
   service,
   pageType,
 }) =>
-  describe(`Running testsForAMPOnly for ${service} ${pageType}`, () => {
```

Comment:
Don't remove this describe, it's very useful for debugging. Nest the describe inside it.